doc/man/man3/notcurses_tabbed.3.md

```diff
@@ -0,0 +1,159 @@
+% notcurses_tabbed(3)
```

Comment:
yesssssss. also please update `index.html` in the parent folder to include this in the `3` section. should be simple.